parsing/blocks/blockparser/common.go

```diff
@@ -35,6 +35,11 @@ func convertIndexerTxsToParserTxs(txs []indexer.Transaction) []Transaction {
 }
 
 func convertIndexerBlockToParserBlock(block indexer.Block, hash string) Block {
+	// TODO: Check why we discard block.Hash.
```

Comment:
We don't really have a body hash.
The block body is a list of miniblocks, and for each of the miniblocks we store the hashes in the miniblock headers (in the block header)

Reply:
Fixed (removed TODO).